JSDemos/Demos/TabPanel/Overview/description.md

```diff
@@ -1,5 +1,28 @@
-<<<<<<< HEAD
-The TabPanel is&nbsp;a&nbsp;UI component consisting of&nbsp;the Tabs and the MultiView UI components. It&nbsp;automatically synchronizes the selected tab with the currently displayed view and vice versa. In&nbsp;this demo, the appearance of&nbsp;views and tabs is&nbsp;defined by&nbsp;custom templates.
-=======
-The TabPanel is&nbsp;a&nbsp;UI component consisting of&nbsp;the Tabs and the MultiView components. It&nbsp;automatically synchronizes the selected tab with the currently displayed view and vice versa. In&nbsp;this demo, the appearance of&nbsp;views and tabs is&nbsp;defined by&nbsp;custom templates.
->>>>>>> e74b8529 (Cut off "UI" from "UI component" is certain cases (#612))
+The [TabPanel](/Documentation/ApiReference/UI_Components/dxTabPanel/) UI component consists of the [Tabs](/Documentation/ApiReference/UI_Components/dxTabs/) and [MultiView](/Documentation/ApiReference/UI_Components/dxMultiView/) components. TabPanel automatically synchronizes the selected tab with the currently displayed view and vice versa.
+
+### Generate Similar Tabs Based on a Data Source       
+
+To generate similar tabs and views, bind the TabPanel to data. Use the [items[]](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/items/) or [dataSource](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#dataSource) property to do this. Both these properties can work with local arrays, but **dataSource** also accepts a [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/) object. You can use this object if you need to process the local array or fetch the array from a remote data source. In this demo, the **dataSource** property is set to a local array.
+
+Each object in the **items[]** or **dataSource** array can contain predefined fields, such as **title** or **icon** (see the [items[]](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/items/) section for the full list). The TabPanel automatically recognizes these fields and creates default tab and view appearance based on them. However, if your data objects contain custom fields, you need to specify the [itemTitleTemplate](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#itemTitleTemplate) and [itemTemplate](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#itemTemplate) that define how to display the fields in tabs and views. This demo illustrates the latter use case.
+
+### Create Individual Tabs
```

Comment:
I believe these are contraposed to "similar" tabs mentioned earlier. While the "similar" tabs are template-based, these have custom layout, am I right?

Reply:
Please see my comment about 'template-based' above.

Reply:
Yeah, I see. No more questions then.